packages/paste-website/src/component-examples/TimePickerExamples.ts

```diff
@@ -0,0 +1,166 @@
+  return (
+    <>
+      <Label htmlFor={uidTP} required>
+        What time is your workday over?
+      </Label>
+      <TimePicker required id={uidTP} aria-describedby={uidHT} {...props} />
+      <HelpText id={uidHT}>Select a time.</HelpText>
+    </>
+  );
+};
+render(<RequiredTimePicker/>)
+`.trim();
+
+export const TimeRangePicker = `
+const TimeRangePicker = (props) => {
+  const [startTime, setStartTime] = React.useState('');
+  const startUidTP = useUID();
+  const endUidTP = useUID();
+  return (
+    <Stack orientation="horizontal" spacing="space80">
+      <Box>
+        <Label htmlFor={startUidTP}>Start time</Label>
+        <TimePicker id={startUidTP} onChange={(evt) => setStartTime(evt.target.value)} {...props} />
+      </Box>
+      <Box>
+        <Label htmlFor={endUidTP}>End time</Label>
+        <TimePicker id={endUidTP} min={startTime} {...props} />
+      </Box>
+    </Stack>
+  );
+};
+render(<TimeRangePicker/>)
+
+`.trim();
+
+export const OnChangePicker = `
+const OnChangePicker = (props) => {
+  const [value, setValue] = React.useState('');
+  const [timeFormat, setTimeFormat] = React.useState('HH:mm');
+  const uidTP = useUID();
+  const uidHT = useUID();
+  const timeFormatOptions = ['hh:mm aa', 'hh:mm a', 'HH:mm'];
+  const handleChange = (val, format) => {
+    setValue(formatReturnTime(val, format));
+    return value;
+  };
+  return (
+    <>
+      <Box marginBottom="space60">
+        <Combobox
+          items={timeFormatOptions}
+          labelText="Return time format:"
+          onInputValueChange={({inputValue}) => {
+            if (inputValue !== undefined) setTimeFormat(inputValue);
+          }}
+        />
+      </Box>
+      <Label htmlFor={uidTP}>What time were you born?</Label>
+      <TimePicker
+        id={uidTP}
+        aria-describedby={uidHT}
+        onChange={(evt) => {
+          handleChange(evt.target.value, timeFormat);
+        }}
+        {...props}
+      />
+      <HelpText id={uidHT}>Select a time above.</HelpText>
+      <Box marginTop="space60">
+        {' '}
+        <b>Return value:</b> {value}{' '}
+      </Box>
+    </>
+  );
+};
+render(<OnChangePicker/>)
+`.trim();
+
+export const HasErrorPicker = `
+const HasErrorPicker = (props) => {
+  const uidTP = useUID();
+  const uidHT = useUID();
+  return (
+    <>
+      <Label htmlFor={uidTP}>When does your workday start?</Label>
+      <TimePicker id={uidTP} aria-describedby={uidHT} hasError {...props} />
+      <HelpText id={uidHT} variant="error">
+        Select a time.
+      </HelpText>
+    </>
+  );
+};
+render(<HasErrorPicker/>)
+`.trim();
+
+export const DisabledPicker = `
+const DisabledPicker = (props) => {
+  const uidTP = useUID();
+  const uidHT = useUID();
+  return (
+    <>
+      <Label htmlFor={uidTP} disabled>
+        Set a curfew for your child
+      </Label>
+      <TimePicker id={uidTP} aria-describedby={uidHT} disabled {...props} />
+      <HelpText id={uidHT}>Choose a time</HelpText>
```

Comment:
```suggestion
      <HelpText id={uidHT}>Choose a time.</HelpText>
```
just to match the other help texts ;)